plonetheme/barceloneta/theme/backend.xml

```diff
@@ -47,7 +47,7 @@
 
     <!-- We can't control the bundle from here due to include. Just hard code -->
     <after css:theme-children="head">
-      <link href="++theme++barceloneta/css/barceloneta.css"
+      <link href="{$portal_url}/++theme++barceloneta/css/barceloneta.css"
```

Comment:
I tried it and checked it again, you've to use {$portal_url} otherwise it prints $portal_url/++theme++barceloneta/css/barceloneta.css instead of <the portal url>/++theme++barceloneta/css/barceloneta.css

Reply:
OK, then it's fine! Go and merge :)

Reply:
You beat me to it, was checking... @yurj is indeed correct :) I have it implemented on a Plone 5.2 website, like this `<a href="{$portal_url}/{$language}/manufactory/distribution-partners">Distribution Partners</a>`